app/src/main/java/com/example/pangea/FacebookHandler.kt

```diff
@@ -0,0 +1,84 @@
+package com.example.pangea
+
+import android.content.Context
+import android.util.Log
+import android.widget.Button
+import com.facebook.*
+import com.facebook.login.LoginManager
+import com.facebook.login.LoginResult
+import com.facebook.login.widget.LoginButton
+
+
+class FacebookHandler(private val context: Context)
+{
+    private var facebook: FacebookSdk? = null;
+    var callbackManager: CallbackManager = CallbackManager.Factory.create();
+
+    fun getAccessToken(): AccessToken

```

Comment:
This function seems to be unused, either remove it or use this function to get the access token.